tools/experiments/experiments.js

```diff
@@ -86,6 +86,11 @@ const EXPERIMENTS = [
     name: 'AMP carousel using horizontal scroll',
     spec: '',
   },
+  {
+    id: 'amp-app-banner',
+    name: 'Shows a native app install/open banner.',
```

Comment:
Please indicate that this isn't going to show the platform native app banner (from Apple or Android) but a banner created by the publisher.